src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcHandle.cs

```diff
@@ -21,8 +21,6 @@ protected OdbcHandle(ODBC32.SQL_HANDLE handleType, OdbcHandle parentHandle) : ba
             bool mustRelease = false;
             ODBC32.RetCode retcode = ODBC32.RetCode.SUCCESS;
 
-            // using ConstrainedRegions to make the native ODBC call and AddRef the parent
-            RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.